gourmet/plugins/python_shell/ipython_view.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
Can't we just remove this line? IMO there's no reason of using shebangs nowadays because of too many ways of storing `python` binaries?